### ISSUE TITLE
Fix for classic status bar transition bug

### ISSUE THREAD

**Issue by duraidabdul** [andrewwiik/LittleX#73]
With the iPhone X status bar is disabled in preferences, the classic status bar has a glitch when leaving control centre. This bug was fixed in HomeGesture, here's the fix

%hook CCUIOverlayStatusBarPresentationProvider
- (void)_addHeaderContentTransformAnimationToBatch:(id)arg1 transitionState:(id)arg2 {
	return;
}
%end

**Comment by mactavish14** [comment]
Does this fix remove the status bar from the cc? Because in HomeGesture, it removes it.

**Comment by duraidabdul** [comment]
It removes it from cc with a nice fade animation, which fixes the entire disappearing bug

**Comment by mactavish14** [comment]
Ah ok. What I really wanted was for the control center to retain the iPhone X status bar, but everywhere else has the normal status bar, and for the animation to be smooth like this. Not sure if that is possible. :/

**Comment by duraidabdul** [comment]
That would be cool, but for now this would be by far the best solution for the bug

**Comment by andrewwiik** [comment]
It would be really hard to make a nice transition between the two as they are so radically different

**Comment by mactavish14** [comment]
Fair enough. Could you use the code suggested above, so when the status bar toggle is off, there us no status bar shown in the control centre - this will eliminate the glitch with the transition. Works really nice in Home Gesture.
